### PR TITLE
Update installation to keep a link to libgfortran.so on deps/

### PR DESCRIPTION
**Pull request by abelsiqueira**
Related to https://github.com/abelsiqueira/linux-cutest/issues/11

### CODE REVIEW

**Comment by abelsiqueira** [comment]
@dpo Does this work for your server?

**Comment by dpo** [comment]
I can't try now as I'm traveling. I'll have to do it in a few days. If it works on TravisCI, then it should be ok. However, shouldn't we use an absolute path instead of `../../` in the location of `libgfortran.so`? Something like `joinpath(dirname(@__FILE__), "..", "usr", "lib", "libgfortran.so")`?!

File: src/CUTEst.jl [hunk 1]
```diff
@@ -23,6 +23,7 @@ const automat = "AUTOMAT.d";
 const funit   = convert(Int32, 42);
 @static is_apple() ? (const linker = "gfortran") : (const linker = "ld")
 @static is_apple() ? (const sh_flags = ["-dynamiclib", "-undefined", "dynamic_lookup"]) : (const sh_flags = ["-shared"]);
+@static is_apple() ? (const libgfortran = []) : (const libgfortran = [joinpath(@__DIR__, "..", "deps", "usr", "lib", "libgfortran.so")])
```
Comment (dpo) [inline src/CUTEst.jl]: I was testing this and got distracted by #144. Is the `.so` intentional here, or should it be `$(Libdl.dlext)`?

Comment (abelsiqueira) [reply src/CUTEst.jl]: Intentional, because it's specific for Linux, and also because a .so is created by linux-cutest.

**Comment by dpo** [comment]
Thanks.

**Pull request merged**